CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx

```diff
@@ -88,15 +93,60 @@ export default function CreateReport({ onDone }) {
           onDone()
         } else {
           console.error("Files failed to upload!")
+          setShowError(true);
+          const errorMessage = await response.text();
+          setErrorMessage(errorMessage);
         }
       } catch (e) {
         console.error("Error Creating Report - " + e)
       }
     }
   }
 
+  // I am not really sure if I should make a separate component for the Error Popup or not, but for now im doing this
+  const Error = ({ message }) => (
+    <>
+      {/* Overlay div */}
+      <div style={{
+        position: 'fixed',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0,
+        backgroundColor: 'rgba(0, 0, 0, 0.5)',
+        zIndex: 50,
+      }} />
+  
+      {/* Popup div */}
+      <div style={{
+        position: 'fixed',
+        top: '50%', 
+        left: '50%', 
+        transform: 'translate(-50%, -50%)',
+        background: 'white',
+        padding: '20px',
+        zIndex: 100,
+        width: 'auto',
+        textAlign: 'center',
+        borderRadius: '5px',
+      }}>
```

Comment:
We should be using tailwind instead of regular css here

Reply:
fixing now